Makefile

```diff
@@ -18,21 +20,18 @@ GODEP := $(GOPATH)/bin/godep
 $(GODEP):
 	go get -u github.com/tools/godep
 
-build: test
+build:
 	go build -o bin/$(EXECUTABLE) $(PKG)
 
 test: $(PKGS)
 
 $(PKGS): $(GOPATH)/bin/golint
-	@gofmt -w=true $(GOPATH)/src/$@*/**.go
-ifneq ($(NOLINT),1)
-	@echo "LINTING..."
-	@$(GOLINT) $(GOPATH)/src/$@*/**.go
-	@echo ""
-endif
-	@echo "TESTING..."
-	@go test $@ -test.v
-	@echo ""
+	@echo "FORMATTING"
+	@gofmt -l=true $(GOPATH)/src/$@*/*.go
```

Comment:
The `-l` flag lists all files in violation, `FGT` will make that return an error so that the build fails.
Previously we could have code on master that didn't follow `gofmt`.